### DISCUSSION TITLE
show region names on plotmarkers

### DISCUSSION THREAD

**Discussion by yamihn** [GreenleafLab/ArchR#1590] [Questions / Documentation] [answered]
hi, is it possible to show the names of some regions on the maplot using plotmarkers?
I did it manually as example, and I'm asking if there is an option to do it automatically.
![Schermata da 2022-08-26 13-19-27](https://user-images.githubusercontent.com/109409803/186892920-e49b54d6-7aa2-4d18-a4d6-310c84ccb1aa.png)

**Comment by rcorces** [comment, marked as answer]
no this is not available. you are welcome to make your own plots manually from the data stored in the SummarizedExperiment.

**Comment by yamihn** [comment]
Hello @mitchTwoTimes, I used ggplot.

meta_data= markertest@meta.data)
list_data= markertest@***.***@list@lista.data)
new_df= cbind(meta_data, list_data)
View(new_df)
new_df$x <- "Log2FC"

colnames(new_df)[5] <- "Log2FC"
colnames(new_df)[6] <- "Mean"
colnames(new_df)[7] <- "FDR"
colnames(new_df)[8] <- "Pval"
colnames(new_df)[9]<- "MeanDiff"
colnames(new_df)[10] <- "AUC"
colnames(new_df)[11] <- "MeanBGD"

df_filter= dplyr::filter(new_df, FDR < 0.1, abs(Log2FC) > 1)
df_filter_mean  = dplyr::filter(df_filter, Mean > 0.40000000)

new_df$region <- paste(new_df$seqnames, new_df$start, new_df$end, sep = "-")
df_filter$region <- paste(df_filter$seqnames, df_filter$start,
df_filter$end, sep = "-")
df_filter_mean$region <- paste(df_filter_mean$seqnames,
df_filter_mean$start, df_filter_mean$end, sep = "-")

new_df$color <- "NA"
new_df$color <- ifelse((new_df$FDR < 0.1 & abs(new_df$Log2FC) > 1), "red",
"grey50")
new_df$DE <- ifelse((new_df$FDR < 0.1 & abs(new_df$Log2FC) > 1), "DE",
"NOTDE")



ggplot(new_df,
       aes(x=Mean, y=Log2FC)) +
  geom_point(colour=new_df$color, aes(shape= DE)) +
  geom_hline(yintercept = 0) +
  geom_hline(yintercept = 1, linetype = "dotted", size = 1) +
  geom_hline(yintercept = -1, linetype = "dotted", size = 1) +
  ggtitle("MA Plot") + ggrepel::geom_label_repel(aes(label = region), data
= df_filter_mean, col= "Blue", alpha= 0.6)
